.github/actions/sonar-scanner/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Sonar Scanner'
+description: 'Run Sonar Scanner to load JaCoCo report on SonarCloud'
+inputs:
+  sonar_token:
+    description: 'SonarCloud token'
+    required: true
+  aggregate-report-path:
+    description: 'Path of the aggregated JaCoCo report'
+    required: true
+  project_key:
+    description: 'SonarCloud project key'
+    required: true
+
+
+runs:
+  using: composite
+  steps:
+    - name: Load JaCoCo report on SonarCloud
+      env:
+        SONAR_TOKEN: ${{ inputs.sonar_token }}
+      shell: bash
+      run: |
+        mvn sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.organization=alfresco -Dsonar.projectKey=${{ inputs.project_key}} -Dsonar.coverage.jacoco.xmlReportPaths=${{ inputs.aggregate-report-path }}
```

Comment:
How about adding also sonar:sonar, url, and organization configurable with inputs but with default values for them? @alxgomz

Reply:
I agree on URL and organisation, but sonar: sonar is the only command that can be used to do what we need